Samples/MapScenarios/MapScenarios/Scenarios/WalkTime/WalkTimeViewController.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  TimeTraveledViewController.swift
+//  MapScenarios
+//
+//  Created by Xiangwei Wang on 2/1/19.
+//  Copyright © 2019 Patrick Dunshee. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import PWCore
+import PWMapKit
+
+class WalkTimeViewController: TurnByTurnViewController {
```

Comment:
I'd still rather not have WalkTimeVC be a subclass of TurnByTurnVC -- I just refactored TBT to be a small piece of code to add to the VC, and you can't mix and match features when subclasses as easily

Reply:
What I think was to separate features in each demo VC, then developers can get what they cares about when opening the specific demo VC.

Reply:
Patrick, I think we talked about this before about doing a better architected sample. We could use MVP architecture and have a common presenter that is used by all of the VCs. This way all of the logic moves into the presenter and we show a good way to use the SDK. This may be outside of the scope of this release but something to consider. I can get the ball rolling pretty quick. If you want to we can discuss offline about how it would look. But I do agree with Xiangwei that we should separate the logic from the VC best we can. I think MVP can help us with that.